prov/efa/src/rdm/rxr_env.c

```diff
@@ -83,6 +82,12 @@ void rxr_env_param_get(void)
 		abort();
 	};
 
+	if (getenv("FI_EFA_MTU_SIZE")) {
+		fprintf(stderr,
+			"FI_EFA_MTU_SIZE env variable detected! The use of this variable has been deprecated and as such execution cannot proceed.\n");
+		abort();
```

Comment:
Do we have to abort if the env variable is set?

Reply:
Yes. that is what we did for other deprecated env vars.

Reply:
I was wondering why we could make it customized by envs.

Reply:
> I was wondering why we could make it customized by envs.

It was introduced back when RxR provider was an utility provider that operates on another RDM provider.